addons/cppcheckdata.py

```diff
@@ -1671,19 +1671,19 @@ def reportError(location, severity, message, addon, errorId, extra='', columnOve
                 'extra': extra}
         sys.stdout.write(json.dumps(msg) + '\n')
     else:
-        if is_suppressed(location, message, '%s-%s' % (addon, errorId)):
+        if is_suppressed(location, message, '{}-{}'.format(addon, errorId)):
             return
         loc = '[%s:%i]' % (location.file, location.linenr)
         if len(extra) > 0:
             message += ' (' + extra + ')'
-        sys.stderr.write('%s (%s) %s [%s-%s]\n' % (loc, severity, message, addon, errorId))
+        sys.stderr.write('{} ({}) {} [{}-{}]\n'.format(loc, severity, message, addon, errorId))
         global EXIT_CODE
         EXIT_CODE = 1
 
 def reportSummary(dumpfile, summary_type, summary_data):
     # dumpfile ends with ".dump"
     ctu_info_file = dumpfile[:-4] + "ctu-info"
-    with open(ctu_info_file, 'at') as f:
+    with open(ctu_info_file, 'a') as f:
```

Comment:
Same as above. "text" vs "binary" might be important in some cases so that should be explicitly stated.